### ISSUE TITLE
Corrupted Barricades.dat

### ISSUE THREAD

**Issue by Sebosstian** [SmartlyDressedGames/Unturned#35]
We're experiencing a problem with our server where our barricades.dat file causes the server to be unplayable. Users try to log in but get stuck on the loadingscreen when connecting to the server.

I've deleted all plugins and all dat files and when i place them back 1 by 1 the problem reappears when putting back the barricades file.

We've had the exact same issue a couple of months ago on a different map (first PEI now Washington)
The file is not more than 700kb in size and I can send you it for research.

**Comment by cartman-2000** [comment]
Unturned bug. 

Try destroying all the cases/racks on the server, yesterday's update corrupted all the containers on the server, I had the problem on some of my chars until I wiped them.

**Comment by Sebosstian** [comment]
Hmmm is that the same issue where some people found their crates full of flares?

Anyway, I think all of the maps (on 2 different servers) were at some point loaded on 11.0 and since these servers don't make backups i guess we need to wipe it then

**Comment by Sebosstian** [comment]
And i can't log into the Bugged server to destroy content because it doesn't load anymore :(

**Comment by cartman-2000** [comment]
Yes. You may have to use a different char or reset the char until you get loaded in an area of the map that doesn't auto dc you.
